generators/client/templates/react/package.json.ejs

```diff
@@ -104,7 +104,7 @@ limitations under the License.
     "css-loader": "3.5.3",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.11.0",
-    "eslint-loader": "4.0.2",
+    "eslint-webpack-plugin": "2.1.0",
```

Comment:
React builds are failing. Probable reason: `eslint-webpack-plugin` version 2 minimal supported ESLint version is 7 but React is still on version 6.

Reply:
Is the upgrade easy ? I can try to upgrade if you want

Reply:
@wmarques Better to separate into 2 PRs?

Reply:
For Angular I choosed this strategy for ESLint upgrade:
* update ESLint dependencies + turn off all violated rules in the latest ESLint dependencies
* rule by rule try to resolve errors

ATM 4 PR-s done for that: #12828, #12835, #12845, #12889

I agree that ESLint dependencies update in React should be a separate PR (or PR-s).

Another option for migrating to `eslint-webpack-plugin` in React side is to use version 1.0.0 which should be ESLint 6 compatible.

Reply:
Ok I'm trying to upgrade and I'll see how it goes :) If there is too much I'll stick with the 1.0.0

Reply:
@kaidohallik I successfully upgraded eslint related deps by switching some rules to off unfortunately :(